src/components/BooksQuery.tsx

```diff
@@ -37,14 +37,10 @@ const BooksQuery = () => {
               <div className="info">{book.volumeInfo.title}</div>
             </div>
 
-            {book.volumeInfo.authors?.map((author) => (
-              <p key={author} className="book-authors">
-                <div className="id">
-                  <strong>Authors :</strong>
-                </div>
-                <div className="info">{author}</div>
-              </p>
-            ))}
+            <div className="book-authors">
+              Author{book.volumeInfo.authors.length > 1 && 's'}:
```

Comment:
Every field except the author field is surrounded with a strong tag and a space before the `:`

**Consistently is important. Please remember to be consistent with your code and UI**

```tsx
<strong>Author{book.volumeInfo.authors.length > 1 && 's'}</strong> : {book.volumeInfo.authors.join(', ')}
```